man/man4/zfs.4

```diff
@@ -724,6 +722,30 @@ Number ARC headers to evict per sub-list before proceeding to another sub-list.
 This batch-style operation prevents entire sub-lists from being evicted at once
 but comes at a cost of additional unlocking and locking.
 .
+.It Sy zfs_arc_evict_threads Ns = Ns Sy 1 Pq int
+Controls the number of ARC eviction threads to be used.
+.Pp
+When set to 0, the parallel arc eviction is disabled.
+Only one thread will be used to evict from ARC.
+.Pp
+When set to 1, ZFS will compute the number of required eviction threads
+depending on the number of CPU cores (ncpu_max).
+The minimum number of threads is 1 and applies to systems from 1 to 5 CPU cores.
+Systems with 6 CPU cores get 2 eviction threads.
+ZFS on systems larger than that uses log2 of the CPU count
+plus one for each 64 CPUs.
+This way the number of eviction threads scales up more on high CPU counts.
+Currently, ZFS will not scale automatically beyond 16 threads.
+.Pp
+When set to a value greater than 1, the value will be used as an exact number
+of eviction threads, but no more than the number of allocated eviction threads.
+The number of allocated eviction threads is set in arc_init() and depends
+on the zfs_arc_evict_threads value.
```

Comment:
Style checker complains on `arc_init()` function used before markup.  You could fix that, but I would rephrase it as:
```suggestion
When set to a value greater than 1, the value will be used as an exact number
of eviction threads.
If changed live, it will be limited by number of threads allocated on module
load.
```